hooks/conftest.py

```diff
@@ -62,6 +64,7 @@ def user_filesystem(
     # │           ├── other
     # │           └── article
     # │               ├── article-cls-in-spm.cls
+    # │               ├── bib-in-project.bib
```

Comment:
is there a reason this is not called `bib-in-spm` for consistency? The implication is that it came from the spm template, which is true.